crates/consensus-logic/src/duty/worker.rs

```diff
@@ -410,7 +410,7 @@ fn perform_duty<D: Database, E: ExecEngineCtl>(
             Ok(())
         }
         Duty::CommitBatch(data) => {
-            info!(data = ?data, "commit batch");
+            info!(data = ?data, "commit batch duty");
```

Comment:
`?data`

But really we should not print the contents here, that's unnecessarily verbose, especially for an `info!` log.

Reply:
True. Just logged the idx.